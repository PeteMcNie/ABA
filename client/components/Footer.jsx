import React from 'react'

const Footer = () => {
  const copyright = '2020 ABA'
  const authors = 'A.Walker-Cochrane - J.Alipate - Z.Arnold - P.McNie'

  return (
    <div className='footer'>
      <p>{authors}</p>
      <p>&copy; {copyright}</p>
    </div>
  )
}

export default Footer
