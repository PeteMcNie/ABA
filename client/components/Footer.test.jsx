import React from 'react'
import { render } from '@testing-library/react'
// import '@testing-library/jest-dom'

import Footer from './Footer'

test('component to match snapshot', () => {
  const { container } = render(<Footer />)
  expect(container).toMatchSnapshot()
})
