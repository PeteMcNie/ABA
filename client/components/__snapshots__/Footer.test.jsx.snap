// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component to match snapshot 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="socials"
    >
      <img
        aria-label="Facebook link"
        src="./icons/facebook.svg"
      />
      <img
        aria-label="Github link"
        src="./icons/github.svg"
      />
      <img
        aria-label="Instagram link"
        src="./icons/instagram.svg"
      />
    </div>
    <p>
      A.Walker-Cochrane - J.Alipate - Z.Arnold - P.McNie
    </p>
    <p>
      Â© 
      2020 ABA
    </p>
  </div>
</div>
`;
